### ISSUE TITLE
The new version of uTorrent doesn't work with NVDA

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#1296]
**Reported by KevanGC on 2011-01-03 00:24**
There's a new version of uTorrent out - uTorrent 2.2.

When you first start uTorrent, it shows a message box or something that NVDA doesn't work with. The only way to get around that is to press CTRL+p, this brings up the uTorrent Preferences dialog. Once you're in the dialog, just press Escape, and you get back to uTorrent and skip the message box.
You should fix NVDA with uTorrent.

**Comment by ehollig** [comment]
Utorrent has changed quite a bit over the years. Is this still the case?

**Comment by leonardder** [comment]
This is an old issue. I'm going to close this. If this issue is still relevant, please create a new one and fill in the template with the relevant details.
